docs/src/terminology.md

```diff
@@ -46,7 +46,7 @@ A set of [validators](terminology.md#validator) maintaining a single [ledger](te
 
 ## confirmation time
 
-The wallclock duration between a [leader](terminology.md#leader) creating a [tick entry](terminology.md#tick) and creating a [confirmed block](terminology.md#confirmed-block).
+The elapsed time between the current leader adding a transaction to an entry, and the block that it was placed in receiving a supermajority of votes from the cluster’s validators.
```

Comment:
This seems better. @jstarry, what do you think?

Reply:
Yeah I like it better too!